### PR TITLE
Project Summary and Logo

### PR DESCRIPTION
**Pull request by leonardoz15**
# Description of Pull Request

This pull request adds a project summary and custom made logo to the top of the `README.md`

This also adds the logo file, `gatorgrouper_logo.svg` in the docs directory for now.

Fixes [#114](https://github.com/GatorEducator/gatorgrouper/issues/114)

## Type of Change

Please describe the pull request as one of the following: 

- [ ] Bug fix
- [ ] Breaking change
- [ ] New feature
- [x] Documentation update
- [ ] Other (Please specify any helpful information below)

## Tags

This pull request is made by @GatorEducator/team-carson 

@leonardoz15 is responsible for writing the project summary
@szklenskij is responsible for creating the logo

Please review the `README.md` and suggest changes!

### CODE REVIEW

**Comment by leonardoz15** [comment]
I renamed the `docs` directory to `.github` to match the other documentation updates in PR [#189](https://github.com/GatorEducator/gatorgrouper/pull/189) as requested. Ready to merge whenever.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,26 @@
 # GatorGrouper
 
+![logo](docs/gatorgrouper_logo.svg "GatorGrouper")
```
Comment (Lancasterwu) [inline README.md]: Shouldn't this be `logo`?

Comment (leonardoz15) [reply README.md]: Thanks for finding that. I just fixed it.

**Comment by Lancasterwu** [comment]
Hi @Michionlion, I think this PR is ready to be merged.

**Pull request merged**